components/layout/TopBar.tsx

```diff
@@ -337,13 +320,8 @@ export const TopBar: FC<TopBarProps> = ({ openMobileCb }) => {
             {/* Profile dropdown */}
             <Dropdown items={dropdownItems} position='left' divider={true} className='pl-3'>
               <span className='sr-only'>Open user menu</span>
-              <Avatar
-                rounded='full'
-                src={avatar}
-                placeholderType='person'
-                size='small'
-                status={mainPresence || 'offline'}
-              />
+              <Avatar rounded='full' src={avatar} placeholderType='person' size='small' />
+              <StatusDot status={mainPresence} className='absolute bottom-0 right-0 '></StatusDot>
```

Comment:
I think <`StatusDot`> should be placed inside <`Avatar`>. Let's pass `status` to `Avatar`